tools/ast_gen.cpp

```diff
@@ -2,6 +2,8 @@
 //
 // SPDX-FileCopyrightText: Copyright (c) assignUser
 #include <algorithm>
+#include <bits/ranges_algo.h>
```

Comment:
nit: not needed / internal
```suggestion
#include <bits/ranges_algo.h>
```

Reply:
huh. I blame clangd^^